core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
@jsimsa I support this idea. And I think it could be a separate enum Type to be more type safe.